bin/install_requirements_linux

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+RUBY_VERSION='2.3.1'
+NODE_VERSION='6.2.1'
+
+RED='\033[0;31m'
+PURPLE='\033[0;35m'
+RESET='\033[0m'
+script="$(basename $0)"
+
+log() {
+  echo -e "${PURPLE}${script}:${RESET} $1"
+}
+
+fail() {
+  log -e "${RED}${script} failed:${RESET} $1" >&2
+  exit 1
+}
+
+cd "$( dirname "$0" )/../requirements"
+requirements_dir="$( pwd )"
+
+log "\n\n====> Linux Development Setup <===="
+log "\nEach group of packages will be install one at a time. Follow the prompts to continue.\n"
+read -p "Press enter to continue"
+clear
+
+# Updates package lists
+sudo apt-get update
+# Updates packages to newest version
+sudo apt-get upgrade
+
+log "\n\n====> Package upgrade complete <====\n"
+read -p "Press enter to continue"
+clear
+
+## Git
+sudo apt-get install git
+git config --global color.ui true
+git config --global user.name "your name here"
+git config --global user.email your_email_here
+
+# Git Completion
+git_url="https://raw.githubusercontent.com/git/git/master/contrib/completion"
+curl "$git_url/git-completion.bash" > "$HOME/.git-completion.bash"
+curl "$git_url/git-prompt.sh" > "$HOME/.git-prompt.sh"
+log "Git completion installed."
+
+log "\n\n====> Git and Git Completion install complete <====\n"
+read -p "Press enter to continue"
+clear
+
+## Dependencies
+sudo apt-get install autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm3 libgdbm-dev
+sudo apt-get update
+
+log "\n\n====> Node + Webpack install complete <====\n"
+read -p "Press enter to continue"
+clear
+
+## Postgresql
+
+sudo apt-get install postgresql postgresql-contrib
+# creates admin role in psql for current user
+sudo -u postgres createuser "$USER" -s
+
+log "\n\n====> Postgresql install complete <====\n"
+read -p "Press enter to continue"
+clear
+
+## Rbenv
+
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+export PATH="$HOME/.rbenv/bin:$PATH"
+export eval "$(rbenv init -)"
+source ~/.bashrc
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+
+log "\n\n====> Rbenv install complete <====\n"
+read -p "Press enter to continue"
+clear
+
+## Ruby
```

Comment:
Everything from here down should be shared with the mac installer, except maybe installing NVM